.github/workflows/call-spell-check.yml

```diff
@@ -18,4 +18,7 @@ permissions: {}
 
 jobs:
   call-workflow:
-    uses: nmfs-ost/ghactions4r/.github/workflows/spell-check.yml@main
\ No newline at end of file
+    uses: nmfs-ost/ghactions4r/.github/workflows/spell-check.yml@main
+    with:
+      spell_check_additional_files: true
+      spell_check_report_level: "error"
```

Comment:
Just noting it should also work without quotations around "error" (I've never tested with quotes but I suspect it works?)

Reply:
Thanks. I removed the quotations.